charts/stable/anonaddy/templates/_secrets.tpl

```diff
@@ -6,8 +6,8 @@
 {{- $secretKey := randAlphaNum 64 -}}
 
  {{- with lookup "v1" "Secret" .Release.Namespace $secretName -}}
-   {{- $appKey = index .data "APP_KEY" | b64dec -}}
-   {{- $secretKey = index .data "ANONADDY_SECRET" | b64dec -}}
+   {{- $appKey = index .data "APP_KEY" | b64enc -}}
```

Comment:
No, 
if the app requires the values to in in a b64encrypted format, you have to DEcrypt here.
and encrypt when adding to the secret bellow.

Reply:
so this PR can be closed then, right?

Reply:
Undo changes here and add `| b64enc` in lines 14 & 15.

---

That's assuming that **both** values need to be in base 64 format when inside the container.